CHANGELOG.md

```diff
@@ -1,3 +1,557 @@
+## [Talos 1.9.0-alpha.1](https://github.com/siderolabs/talos/releases/tag/v1.9.0-alpha.1) (2024-11-08)
+
+Welcome to the v1.9.0-alpha.1 release of Talos!  
+*This is a pre-release of Talos*
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/siderolabs/talos/issues.
+
+### AppArmor
+
+Talos Linux starting with v1.9 will ship with SELinux LSM enabled by default.
+If you need to use AppArmor LSM add the following to the machine configuration:
+
+```yaml
+machine:
+  install:
+     extraKernelArgs:
+      - -selinux
+      - lsm=lockdown,capability,yama,apparmor,bpf
+      - apparmor=1
+```
+
+
+### Auditd
+
+Talos Linux now starts a auditd service by default.
+Logs can be read with `talosctl logs auditd`.
+
+
+### `talosctl cgroups`
+
+The `talosctl cgroups` command has been added to the `talosctl` tool.
+This command allows you to view the cgroup resource consumption and limits for a machine, e.g.
+`talosctl cgroups --preset memory`.
+
+
+### udevd
+
+Talos previously used `udevd` to provide `udevd`, now it uses `systemd-udevd` instead.
+
+
+### Component Updates
+
+Linux: 6.6.59
```

Comment:
this is now rendered as a single line, should have been a list

Reply:
Yep, let's fix it in main for the next bump